chart/templates/_helpers.yaml

```diff
@@ -214,8 +214,10 @@ If release name contains chart name it will be used as a full name.
       value: "repo"
     - name: GIT_SYNC_ADD_USER
       value: "true"
-    - name: GIT_SYNC_WAIT
+    - name: GIT_SYNC_WAIT # Deprecated: use GIT_SYNC_PERIOD
       value: {{ .Values.dags.gitSync.wait | quote }}
+    - name: GIT_SYNC_PERIOD
+      value: {{ .Values.dags.gitSync.period | quote }}
```

Comment:
Since `wait` is used when both are set, we might as well stick with just `wait` here as we can't remove it (yet).

Reply:
What is the advantage of delaying support of the new flag? New deployments can start using it right away and not have to migrate to it at a later date. Existing deployments can migrate during deprecation period to avoid a straight cut-over where the version and the helm values have to all be changes at the same time.

Reply:
Oh, but I do see a problem with my implementation... Maybe a better way to do it is to only include `period`, and just use the value from `wait` in that place if it is given.

Reply:
Why do we need to change the parameter at all? Would not taht be enough (and I think @jedcunningham suggests it) use GIT_SYNC_PERIOD and pass the `.Values.dags.gitSync.wait` to it.

While not super convenient, I do not see big value in changing the name. For most people who do not read the documentation but use autocomplete and such - it will be fine as they likely do not even look at git-sync docs. For those who read the documentation you can explain it..

I think that should be enough?

Or am I missing something?

Reply:
Yeah, that's basically where I was going.

We have to set both env vars so it works regardless of gitsync version. And it'll prefer `WAIT`, so `PERIOD` is never used anyways. We could do some more complicated logic to support both, but I agree sticking with just `wait` in our values seems easiest.